README.md

```diff
@@ -9,4 +134,243 @@ The website for the Civic Tech Index
+`5773ebe..0c86ecd  gh-pages   -> CTI-website-frontend/gh-pages`, you should
+incorporate those changes into your repository before merging or
+rebasing your issue branch. Use the [instructions below](#incorporating-changes-from-upstream)
+to bring your fork up to date with the main repository.
+
+
+### Incorporating changes from upstream
+
+Your fork of this repository on GitHub, and your local clone of that fork, will
+get out of sync with this (upstream) repository from time to time.  (That's what has happend when you see something like "This branch is 1 commit behind CTI-website-frontend:gh-pages" on the github website version of your CTI-website-frontend repository.)
+
+One way to keep your fork up to date with this repository is to follow
+these instruction: [Syncing your fork to the original repository via the browser](https://github.com/KirstieJane/STEMMRoleModels/wiki/Syncing-your-fork-to-the-original-repository-via-the-browser)
+
+You can also update your fork via the local clone of your fork, using
+these instructions. Assuming you have a local clone with remotes
+`upstream` (this repo) and `origin` (your GitHub fork of this repo):
+
+First, you will need to create a local branch which tracks upstream/gh-pages.  You will only need to do this once; you do not need to do this every time you want to incorporate upstream changes. 
+
+Run the following two commands: 
+
+```bash
+git fetch upstream
+git checkout -b upstream-gh-pages --track upstream/gh-pages
+```
+
+If you have already created the branch upstream-gh-pages, the following commands will incorporate upstream changes: 
+
+```bash
+git checkout upstream-gh-pages # Move to the branch you want to merge with. 
+git pull  # This updates your tracking branch to match the gh-pages branch in this repository
+git checkout gh-pages  # Move back to your gn-pages branch
+git merge upstream-gh-pages  # Merge to bring your gh-pages current. 
+```
+If you do all your work on topic branches (as suggested above) and keep gh-pages free of local modifications, this merge should apply cleanly.
+
+Then push the merge changes to your GitHub fork:  
+
+```bash
+git push
+```
+If you go to your online github repository this should remove the message "This branch is x commit behind CTI-website-frontend:gh-pages".
+
+### Incorporating changes into your topic branch
+
+To incorporate these updates from the main GitHub repository into your
+topic branch, you can 'rebase' your branch onto your updated gh-pages
+branch. NOTE you should only rebase if you have never pushed your
+topic branch to GitHub (or shared it with another collaborator).
+
+```bash
+git checkout fix-logo-width-311
+git rebase gh-pages
+```
+
+If you receive warnings about conflicts, abort the rebase with `git
+rebase --abort` and instead merge gh-pages into your branch.
+
+```bash
+git checkout fix-logo-width-311
+git merge gh-pages
+```
+
+## Step 7 Complete the pull request
+
+```bash
+git push --set-upstream origin fix-logo-width-311
```

Comment:
remind them to replace "fix-logo-width-311" with whatever they called their branch

Reply:
There is already a reminder there:
"The name at the end of this command should be the same as the name of the local branch that you created back in step 6, as in the example below: "